morpho5/docs/functionals.md

```diff
@@ -192,18 +192,41 @@ You can also integrate functions that involve fields:
 
 More than one field can be used; they are passed as arguments to the integrand function in the order you supply them to `AreaIntegrand`.
 
-## FloryHuggins
+## Hydrogel
 [tagfloryhuggins]: # (floryhuggins)
+[taghydrogel]: # (hydrogel)
```

Comment:
Looks like the old and new documentations got combined here instead of replacement?